docs/api-docs/partner/subscriptions/foundation-guide.md

```diff
@@ -0,0 +1,231 @@
+> During installation, the app UI prompts the merchant to enter their Stripe account's keys so that it can initiate an OAuth grant code authorization flow that links the merchant's Stripe account to the app's Stripe Connect-enabled account. This arrangement allows the merchant to change the Stripe account they use without breaking the app. It also ensures that the merchant's self-managed Stripe account stores all charge and payment data, including both subscriptions and the shopper's one-time purchases. This architectural pattern dramatically reduces the app's PCI compliance burden. 
+
+### Create Stripe accounts
+
+To get started, do the following steps:
+
+1. To create the Stripe account that simulates your merchant's pre-existing Stripe account, [sign up with Stripe](https://dashboard.stripe.com/register). Your app uses this account's client ID to process subscriber payments and manage subscriptions; give it a merchant-specific name.
+
+2. Sign in to the [Stripe Dashboard](https://dashboard.stripe.com/).
+
+3. From inside the [Stripe Dashboard](https://dashboard.stripe.com/), create and configure a second app-specific Stripe account. The app uses this account's client secret to process subscription payments.
+
+   a. On the left side of your Stripe Dashboard's top menu bar, click the name of your merchant-specific account. In the dropdown menu that appears, select **+ New Account**.
+
+   b. Give the new account a name that clearly marks it as app-specific; the idea is to differentiate it from the merchant-specific payment and subscription management account. This guide uses **BigCommerce App**.
+   
+   ![BigCommerce App](https://storage.googleapis.com/bigcommerce-production-dev-center/images/BigCommerce-app-image.png "The Stripe dashboard's new account dropdown menu")
+
+### Enable Stripe Connect for platforms
+
+**During this instruction sequence, make sure that the left side of your Stripe Dashboard's top menu bar indicates that you're in the app-specific account.**
+
+<!-- theme: info -->
+> #### Test mode
+> You can simulate transactions in test mode to confirm your integration works correctly or enable for production using the toggle button on the upper right. The steps below have been configured for test mode.
+
+1. Click the **Connect** button to enable [Stripe Connect for platforms](https://dashboard.stripe.com/test/connect/accounts/overview). 
+
+2. Click **Get started**.
+
+3. Select **Platform or Marketplace**, then click the **Continue** button at the lower right.
+
+4. Go to [**Settings > Connect settings**](https://dashboard.stripe.com/test/settings/connect), to configure the app-specific account's Stripe Connect settings.
+   
+   a. Under **Test mode client ID**, copy the Test mode client ID. In a later step, you will use the client ID to update `NEXT_PUBLIC_STRIPE_CLIENT_ID` in the .env file.
```

Comment:
rework these steps that start with `Under`. The style guide doesn't love directional references. You could start with describing these settings' sections on the page, or go in another direction entirely.

This section also needs you to get the Connect secret key for later.